testing/refpolicy/APKBUILD

```diff
@@ -0,0 +1,58 @@
+# Maintainer: Tycho Andersen <tycho@docker.com>
+pkgname=refpolicy
+pkgver=20170204
+pkgrel=0
+pkgdesc="SELinux policy reference"
+url="https://github.com/TresysTechnology/refpolicy/wiki"
+arch="noarch"
+license="GPLv2"
+depends=""
+depends_dev=""
+makedepends="$depends_dev checkpolicy python gawk"
+install=""
+subpackages="$pkgname-doc"
+source="https://raw.githubusercontent.com/wiki/TresysTechnology/refpolicy/files/refpolicy-2.$pkgver.tar.bz2"
+builddir="$srcdir/refpolicy"
+
+# refpolicy config
+monolithic=n
+# This causes various distro-specific policies to be built. Since most SELinux
+# policies are written against redhat-based distros, we set that here.
+distro=redhat
```

Comment:
This seems suspect to me.
There is a lot of things different in Alpine that would not mesh well with Redhat's policy definitions.
I think either `debian` or `gentoo` would be a much closer fit.